### DISCUSSION TITLE
I'm going to start working on ...

### DISCUSSION THREAD

**Discussion by togfoxy** [togfoxy/MarsLander#244] [General]
Feel free to post your current work efforts so we can do our best to not double up. Your current work can be bit or small, quick or slow. Let us know what you've start so we know not to do that. :)

**Comment by togfoxy** [comment]
I've got a branch for multiplayer improvements.

Also a branch on a new parachute image and behavior.

**Comment by MadByteDE** [comment]
I started working on the hopefully final bits of restructure as described [here](https://github.com/togfoxy/MarsLander/issues/150#issuecomment-964347119).

And if I get stuck on that mission, I'll also have a branch for the lander-physics to start working on that.

**Comment by togfoxy** [comment]
![Untitled](https://user-images.githubusercontent.com/5599266/141673195-f2cfea23-dd96-4e73-bb0e-63d74a5e0865.png)

Lot's of fun.  :)

**Comment by Philbywhizz** [comment]
I haven't been doing much lua/love coding. What I have been doing however is playing around with github actions on my own lua project. 

I've now go it so that then you run the action it automatically creates a mac osx, win32, win64 and a .love package. If the current commit has a tag on it it will draft up a release with an extract of the changelog.

Once I tidy up the code I might look at adding it to MarsLander for your enjoyment. :)

**Comment by togfoxy** [comment]
That's great stuff. Will be good to see it working.

I'm done with the parachute. I like it.I'm moving on to fixing up those multiplayer loose ends.

**Comment by togfoxy** [comment]
A new settings window. I'll document how to add game options. Actually quite easy.

![image](https://user-images.githubusercontent.com/5599266/142186222-b301d1ef-9343-4af8-a22b-5d60ed2d1f5c.png)
